users/users.md

```diff
@@ -116,6 +116,56 @@ Response
 ```
 ---
 
+## Get a list of user group that a user belongs to
+
+Returns a list of user group a user belongs to.
+
+```apacheconfig
+GET /v1/users/{userId}/usergroups
+```
+
+#### Path parameters
+
+| Name | Type | Description |
+| ------------- |------------- | -------------|
+| userId | `string` | The id of a user to request. |
+
+
+#### Query parameters
+The query parameters are not expected.
+
+#### Request body
+The request body must be empty.
+
+#### Response body
+The successful response contains a [User Group List Item](../user_groups/user_groups_schemas.md/#user-group-list-item).
```

Comment:
response contains a list of items, not just one item

Reply:
done